specification/relay/resource-manager/Microsoft.Relay/stable/2016-07-01/examples/RelayOperations_List.json

```diff
@@ -11,183 +11,163 @@
-              "description": "Get list of HybridConnection Resource Descriptions"
+              "operation": "Get HybridConnection"
             }
           },
           {
             "name": "Microsoft.Relay/namespaces/HybridConnections/Delete",
             "display": {
               "provider": "Microsoft Azure Relay",
               "resource": "HybridConnection",
-              "operation": "Delete HybridConnection",
-              "description": "Operation to delete HybridConnection Resource"
+              "operation": "Delete HybridConnection"
             }
           },
           {
             "name": "Microsoft.Relay/namespaces/HybridConnections/authorizationRules/write",
             "display": {
               "provider": "Microsoft Azure Relay",
               "resource": "HybridConnection AuthorizationRules",
-              "operation": "Create or Update HybridConnection Authorization Rule",
-              "description": "Create HybridConnection Authorization Rules and Update its properties. The Authorization Rules Access Rights, the Primary and Secondary Keys can be updated."
+              "operation": "Create or Update HybridConnection Authorization Rule"
             }
           },
           {
             "name": "Microsoft.Relay/namespaces/HybridConnections/authorizationRules/delete",
             "display": {
               "provider": "Microsoft Azure Relay",
               "resource": "HybridConnection AuthorizationRules",
-              "operation": "Delete HybridConnection Authorization Rules",
-              "description": "Operation to delete HybridConnection Authorization Rules"
+              "operation": "Delete HybridConnection Authorization Rules"
             }
           },
           {
             "name": "Microsoft.Relay/namespaces/HybridConnections/authorizationRules/listkeys/action",
             "display": {
               "provider": "Microsoft Azure Relay",
               "resource": "HybridConnection AuthorizationRules",
-              "operation": "List HybridConnection keys",
-              "description": "Get the Connection String to HybridConnection"
+              "operation": "List HybridConnection keys"
             }
           },
           {
             "name": "Microsoft.Relay/namespaces/WcfRelays/write",
             "display": {
               "provider": "Microsoft Azure Relay",
               "resource": "WcfRelay",
-              "operation": "Create or Update WcfRelay",
-              "description": "Create or Update WcfRelay properties."
+              "operation": "Create or Update WcfRelay"
             }
           },
           {
             "name": "Microsoft.Relay/namespaces/WcfRelays/read",
             "display": {
               "provider": "Microsoft Azure Relay",
               "resource": "WcfRelay",
-              "operation": "Get WcfRelay",
-              "description": "Get list of WcfRelay Resource Descriptions"
+              "operation": "Get WcfRelay"
```

Comment:
why have the descriptions been pulled out here?

Reply:
Description isn't part of the Swagger spec, and was likely added because of a VS Code JSON schema bug that suggest "description" be added to example files. Without this the model validation will fail (see first commit). I can submit it separately, but the build won't pass. Let me know how you want to handle it

Reply:
@praries880 I've opened an issue with some background over here #6277, since it has come up on a few PR 😄 
I'm still good if you want it removed from this PR